backends/p4tools/scripts/tools/cpplint.py

```diff
@@ -0,0 +1,6903 @@
+#!/usr/bin/env python3
```

Comment:
do we really need two copies of cpplint?
I can understand we need two configurations, but we should be able to reuse the tool.